sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -43,6 +43,12 @@ namespace pi {
   using PiMemFlags            = ::pi_mem_flags;
   using PiEvent               = ::pi_event;
   using PiSampler             = ::pi_sampler;
+  using PiImageFormat         = ::pi_image_format;
+  using PiImageDesc           = ::pi_image_desc;
+  using PiImageInfo           = ::pi_image_info;
+  using PiMemObjectType       = ::pi_mem_object_type;
```

Comment:
PiImageInfo & PiMemObjectType are unused? Can we remove these?
At least I would like to get read of all object in memobject

Reply:
They are used in ./include/CL/sycl/detail/image_impl.hpp